### PR TITLE
Update issue templates

### LINKED ISSUE #4469: sunpy logo link broken
The link to "sunpy logo" here: https://sunpy.org/about#acknowledging-sunpy-in-posters-and-talks just redirects to the same page instead of a logo.

### PR DESCRIPTION
**Pull request by nabobalis**
Screenshot from 2020-09-12 13-19-07

I also tried to some issue templates less verbose.

Fixes #4469

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/performance.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: Performance Concerns
```
Comment (Cadair) [inline .github/ISSUE_TEMPLATE/performance.md:2]:
```suggestion
name: Performance Suggestions
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Sunpy Community Chat Room
+    url: https://openastronomy.riot.im/#/room/#sunpy:openastronomy.org
```
Comment (Cadair) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    url: https://openastronomy.element.io/#/room/#sunpy:openastronomy.org
```

**Review by Cadair — COMMENTED** [review]
Can we re-order them, it seems weird that "Anything Else" is at the top?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Sunpy Community Chat Room
```
Comment (Cadair) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
  - name: Sunpy Community Chat
```
?

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (nabobalis) [reply .github/ISSUE_TEMPLATE/config.yml:4]: time to update the readme too

Comment (Cadair) [reply .github/ISSUE_TEMPLATE/config.yml:4]: yeah probably a lot of other places as well.

File: .github/ISSUE_TEMPLATE/performance.md (hunk 1 above)
Comment (nabobalis) [reply .github/ISSUE_TEMPLATE/performance.md]: Fixed.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (nabobalis) [reply .github/ISSUE_TEMPLATE/config.yml]: Fixed.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (nabobalis) [reply .github/ISSUE_TEMPLATE/config.yml]: Fixed.

**Comment by nabobalis** [comment]
> Can we re-order them, it seems weird that "Anything Else" is at the top?

I can try but it looks to be sorted.

**Comment by nabobalis** [comment]
How does Whatever else sound?

**Comment by nabobalis** [comment]
> Can we re-order them, it seems weird that "Anything Else" is at the top?

Done.

**Review by wtbarnes — APPROVED** [review]

**Review by dstansby — APPROVED** [review]

**Pull request merged**